.circleci/config.yml

```diff
@@ -225,7 +264,30 @@ jobs:
           root: /app
           paths:
             - core-<<parameters.language>>
-
+  dev_test_integration:
+    description: Run integration tests against deployed dev service.
+    parameters:
+      <<: *param__working_directory
+      <<: *param__language
+    working_directory: ~/project/<<parameters.working_directory>>
+    executor: python/python
+    steps:
+      - setup-project
+      - run:
+          name: Run integration tests
+          command: CORE_ENDPOINT_URL=https://${CIRCLE_PROJECT_REPONAME}-pr-${CIRCLE_PR_NUMBER}-core-<<parameters.language>>.dev.dialoguecorp.com poetry run python -m rasa_integration_testing ../integration-tests-<<parameters.language>>
+  master_test_integration:
+    description: Run integration tests against deployed service on master.
+    parameters:
+      <<: *param__working_directory
+      <<: *param__language
+    working_directory: ~/project/<<parameters.working_directory>>
+    executor: python/python
+    steps:
+      - setup-project
+      - run:
+          name: Run integration tests
+          command: CORE_ENDPOINT_URL=https://${CIRCLE_PROJECT_REPONAME}-core-<<parameters.language>>.dialoguecorp.com poetry run python -m rasa_integration_testing ../integration-tests-<<parameters.language>>
```

Comment:
Also: specify environement variables using the `environment:` key on the job. See `prepare_core_build` job for example

Reply:
It seems I have to pass the environment variable like this because the URL contains other environment variables which don't get interpreted when the command is run. 🤷

Reply:
Indeed! : https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command

I didn't know that. For clarity, I would use the 
```
echo 'export PATH=/path/to/foo/bin:$PATH' >> $BASH_ENV
```
format then

Reply:
Ok, I'll try that one